### ISSUE TITLE
Some kind of support for adding active effects.

### ISSUE THREAD

**Issue by QckSilverDragon** [Muttley/foundryvtt-fallout#44]
It would be nice if there was an easier way to add active effects to items.

currently you have to use complex data paths but it would be nice for either a preformed list or have a dropdown kind of list of effects to add.

I have seen it done in Savage Worlds system but i dont know how they did it.

**Comment by QckSilverDragon** [comment]
As an example, here is how Savage Worlds uses a "Wizard" for active effects.

![active effect wizard](https://github.com/Muttley/foundryvtt-fallout/assets/4044050/eaf6c6bf-d40d-4dc8-9dec-a48658f3ad62)

**Comment by Muttley** [comment]
Yeah, that looks like something to work towards.  Probably not this next version though.

**Comment by QckSilverDragon** [comment]
Thats fair. Just trying to keep you busy. 😄

**Comment by Unworthypilot** [comment]
in addition to this, can you made affects something that be like an item so they can just be dragged and dropped into the sheet.

**Comment by guver** [comment]
Speaking of which, where can the data paths be found? I'm trying to add an effect for ex. that adds +1 melee damage or +1 Physical resistance but I'm unable to do so.
